lfe-tidier: add argument quick to tidy

MatthieuStigler opened the pull request:
Hi

`tidy.felm` did not have the `quick` option, which was very useful in my case. I made an implementation of it, trying to keep code as in other functions. I did local test, but no global ones (difficult to run R CMD check as so many packages required)... I hope it's fine!?

Thanks!

alexpghayes commented:
This looks great! Can you add some tests for `tibble` output when `quick = TRUE`? Or just construct a `tibble` instead of a `data.frame`?

MatthieuStigler commented:
ok, I did following changes:

- use directly `data_frame`
- added tests with `quick=TRUE` in *test-lfe.R*

I also corrected my own change, where now `quick` takes precedence over `conf.int=TRUE` (i.e. `if (conf.int)` becomes `if (!quick & conf.int)`). This seemed to be the policy in `broom`, am I correct?

Thanks

MatthieuStigler commented:
ok, I did! It seems now it is not passing the test, but seems for a totally different reason?
